lib/fog/openstack/requests/compute/evacuate_server.rb

```diff
@@ -3,12 +3,12 @@ module Compute
     class OpenStack
       class Real
         def evacuate_server(server_id, host, on_shared_storage, admin_password = nil)
+          evacuate_hash = {}
```

Comment:
Also, if `host` and `on_shared_storage` are optional then set them to nil in the method signature, then the method can simply be called as `evacuate_server(id)` instead of `evacuate_server(id, nil, nil)`.